### PR TITLE
Add onClick to Button

### PR DESCRIPTION
**Pull request by MagdaBer**
I got things mixed up and had to open a new pull request.
I added the onClick property to my button component. I made onClick optional. Is this the right approach?

### CODE REVIEW

File: src/app/components/Button/Button.tsx [hunk 1]
```diff
@@ -6,16 +6,12 @@ import Typography from '../Typography/Typography';
 type ButtonProps = {
   children: ReactNode;
   className?: string;
-  onButtonClick: () => void;
+  onClick?: () => void;
```
Comment (marwinburesch) [inline src/app/components/Button/Button.tsx:9]: I wouldn't have this optional, it should always be defined, even in a story in storybook. Just like your `handleSubmit` in your `SearchBar` component

**Review by marwinburesch — COMMENTED** [review]
👍

File: src/app/components/Button/Button.tsx (hunk 1 above)
Comment (MagdaBer) [reply src/app/components/Button/Button.tsx:9]: Could you please check if this is the correct solution.

**Review by marwinburesch — COMMENTED** [review]
`click` click Click CLICK onClick, on point 👍

**Pull request merged**